tests/api/test_store.py

```diff
@@ -188,3 +188,14 @@ async def container_events_task(*args, **kwargs):
     assert resp.status == 200
 
     await _container_events_task
+
+
+@pytest.mark.parametrize("resource", ["store/addons", "addons"])
+async def test_api_store_addons_no_changelog(
+    api_client: TestClient, coresys: CoreSys, store_addon: AddonStore, resource: str
+):
+    """Test /store/addons/{addon}/changelog REST API."""
+    resp = await api_client.get(f"/{resource}/{store_addon.slug}/changelog")
+    assert resp.status == 200
```

Comment:
Maybe just add a comment here to explain this behavior is expected by the frontend? And change it to 404 once frontend is able to handle the errors graciously.

Reply:
Agreed, added a suggestion.